app/AvailablePlugin/EmailAddressWidget/Config/Schema/schema.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" ?>
+<!--
+  COmanage Registry Email Address Widget Plugin Database Schema
+
+  Portions licensed to the University Corporation for Advanced Internet
+  Development, Inc. ("UCAID") under one or more contributor license agreements.
+  See the NOTICE file distributed with this work for additional information
+  regarding copyright ownership.
+
+  UCAID licenses this file to you under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with the
+  License. You may obtain a copy of the License at:
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+-->
+<!--
+  While this schema mostly works with database prefixes, foreign key constraints
+  must be specified in raw SQL, which needs the prefixed table name.
+-->
+<schema version="0.3">
+  <table name="co_email_address_widgets">
+    <field name="id" type="I">
+      <key />
+      <autoincrement />
+    </field>
+    <field name="default_type" type="C" size="128" />
+    <field name="type_hidden" type="L" />
+    <field name="disable_provisioning" type="L" />
+    <field name="verification_validity" type="I" />
+    <field name="co_message_template_id" type="I">
+      <constraint>REFERENCES cm_co_message_templates(id)</constraint>
+    </field>
+    <field name="co_dashboard_widget_id" type="I">
+      <constraint>REFERENCES cm_co_dashboard_widgets(id)</constraint>
+    </field>
+    <field name="created" type="T" />
+    <field name="modified" type="T" />
+
+    <index name="co_email_address_widgets_i1">
+      <col>co_dashboard_widget_id</col>
+      <unique />
+    </index>
+    <index name="co_email_address_widgets_i2">
+      <col>co_message_template_id</col>
+    </index>
+  </table>
+
+
+  <table name="email_address_widget_verifications">
+    <field name="id" type="I">
+      <key />
+      <autoincrement />
+    </field>
+    <field name="email" type="C" size="128" />
+    <field name="type" type="C" size="32" />
+    <field name="token" type="C" size="16" />
+    <field name="co_email_address_widget_id" type="I">
+      <constraint>REFERENCES cm_co_email_address_widgets(id)</constraint>
+    </field>
+    <field name="co_person_id" type="I">
+      <constraint>REFERENCES cm_co_people(id)</constraint>
+    </field>
+    <field name="created" type="T" />
+    <field name="modified" type="T" />
+
+    <index name="email_address_widget_verifications_i1">
+      <col>co_email_address_widget_id</col>
+    </index>
+    <index name="email_address_widget_verifications_i2">
+      <col>co_person_id</col>
+    </index>
```

Comment:
I'm not sure we need either of these indexes, but the one we actually do need (`token`, which is what `calculateImpliedCoId` uses) is missing.